modules/cloudbuild/main.tf

```diff
@@ -277,9 +282,6 @@ resource "google_organization_iam_member" "cloudbuild_serviceusage_consumer" {
   org_id = var.org_id
   role   = "roles/serviceusage.serviceUsageConsumer"
   member = "serviceAccount:${module.cloudbuild_project.project_number}@cloudbuild.gserviceaccount.com"
-  depends_on = [
-    google_project_service.cloudbuild_apis,
```

Comment:
is it safe to remove these? Previously there was race conditions with APIs being enabled & the SA existing in time for permissions to be applied.

Reply:
I didn't run issues  when testing locally. Within PF we already have a dependency on API enablement before returning `project_number` [here](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/4275a8d52d5455cb16f8af5f179ea882a4eaeedb/modules/core_project_factory/outputs.tf#L36). Best approach would be to use the service identity returned [directly](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/4275a8d52d5455cb16f8af5f179ea882a4eaeedb/modules/project_services/outputs.tf#L27). I will open a PR in PF to plumb that through.

Reply:
new PF release will have that output
https://github.com/terraform-google-modules/terraform-google-project-factory/pull/549